docs/messaging/sms/specs/ringcentral_environment.postman_environment.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
Can we untabify this file? 4 spaces per tab is good. 

Also - do we want to embed these files in the guide itself so that they are visible and viewable to developers?

Reply:
This is generated by Postman, so I don't want to change/tamper with it. 

For now, I have added the screenshot, but may be later if it makes sense, we will embed it. I can talk to Craig and do it next week or so. Not critical for this PR.